apps/client/src/pages/create.tsx

```diff
@@ -271,19 +270,20 @@ class Create extends React.Component<CreateProps, CreateState> {
     const cardBody = !this.state.created ? (
       <div>
         <Banner />
-        <p>
-          Elevation of Privilege (EoP) is the easy way to get started and learn
-          threat modeling. It is a card game that developers, architects or
-          security experts can play.
-        </p>
-        <p>
-          To learn more about the game, navigate to the{' '}
-          <Link to="/about">about page</Link>.
-        </p>
-        <small className="text-secondary">
-          To start playing, select the number of players and enter their names.
-        </small>
+        <Button
+          block
+          size="lg"
+          color="primary"
+          onClick={() => (window.location.href = `/random-card`)}
+        >
+          Draw a random card
+        </Button>
         <hr />
+        <p className="space-top">
+          <center>
+            <strong>Or create a new game:</strong>
+          </center>
```

Comment:
This should not be done with HTML elements. It's just styling, so it should just be CSS.

Reply:
Becomes obsolete with above change